src/modality-specific-files/behavioral-experiments.md

```diff
@@ -76,6 +72,141 @@ A guide for using macros can be found at
 -->
 {{ MACROS___make_sidecar_table("beh.BEHInstitutionInformation") }}
 
+## Audio and video recordings
+
+Audio and video recordings of behaving subjects MAY be stored in the `beh` directory
+using the `_audio` and `_video` suffixes respectively.
+These recordings are typically used to capture vocalizations, speech, facial expressions,
+body movements, or other behavioral aspects during experimental tasks or rest periods.
+
+!!! warning "Privacy and personally identifiable information"
+
+    Audio and video recordings of human subjects often contain personally identifiable
+    information (PII) such as faces, voices, and other identifying features.
+    Data curators MUST take special care to ensure compliance with applicable privacy
+    regulations (such as HIPAA in the United States, GDPR in the European Union, or other
+    local data protection laws) when handling these recordings.
+
+    These recordings are generally more suitable for internal use or for sharing
+    non-human subject data, unless appropriate privacy protections are implemented.
+
+### File formats
+
+Audio recordings MUST use one of the following extensions:
+
+-   `.mp3` - MPEG Audio Layer III
+-   `.wav` - Waveform Audio File Format
+
+Video recordings MUST use one of the following extensions:
+
+-   `.mp4` - MPEG-4 Part 14
+-   `.mkv` - Matroska video container
+-   `.avi` - Audio Video Interleave
+
+### Entities
+
+Audio and video files MAY use the following entities:
+
+-   `task` - OPTIONAL for audio and video recordings
+-   `acquisition` - OPTIONAL, can distinguish different recording setups
+-   `run` - OPTIONAL, for multiple recordings with identical parameters
+-   `recording` - OPTIONAL, to differentiate simultaneous recordings from different angles, locations, or devices
+-   `split` - OPTIONAL, for continuous recordings split into multiple files
+
+### Examples
+
+```Text
```

Comment:
see/use MACROS___make_filetree_example here or provide comprehension when such trees are encountered in `Text` blocks, parsed into hierarchical structury,  and then proxied to that MACROS for actual uniform rendering